index.js

```diff
@@ -30,10 +30,11 @@ var colors = {
 
 module.exports = Nanologger
 
-function Nanologger (name) {
-  if (!(this instanceof Nanologger)) return new Nanologger(name)
+function Nanologger (name, colors) {
```

Comment:
I think it might be better to have a generic `opts` object with a `.colors` key instead. That way it feels a bit more in line with a lot of other module APIs